templates/download/shared/_first-boot-setup.html

```diff
@@ -8,11 +8,11 @@ <h2 class="p-stepped-list__title">First boot setup</h2>
       <li class="p-list__item">Press enter then select "Start" to begin configuring your network and an administrator account. Follow the instructions on the screen, you will be asked to configure your network and enter your Ubuntu SSO credentials.</li>
       <li class="p-list__item">
         At the end of the process, you will see your credentials to access your Ubuntu Core machine:
-        <p></p>
+        <br /><br />
```

Comment:
Same comment here as on the other pr, I believe it's best practice to use margin to add spacing here

Reply:
Fixed with the rebase